packages/swc-angular-plugin/src/component_property_visitor/component_property_visitor_test.rs

```diff
@@ -26,6 +26,34 @@ fn test_input() {
     );
 }
 
+#[test]
```

Comment:
just curious, what is the problem that this test detects?
was there a case where services affected the inputs transform?

Reply:
that I try to test problem where #304 described, but looks like problem not here, I can delete that tests